src/monodroid/jni/cpu-arch-detect.cc

```diff
@@ -225,7 +225,7 @@ get_running_on_cpu (unsigned short *running_on_cpu)
 	*running_on_cpu = CPU_KIND_UNKNOWN;
 }
 
-MONO_API void
```

Comment:
Why remove `MONO_API`?  Is this going to become a non-"public" function?

Reply:
Yes, it was moved to `external-api.cc` and it is not expected to be called externally on `libmonodroid.so`